README.md

```diff
@@ -34,25 +34,27 @@ Installation
 If you need to customize **ember-cli-workbox configuration** you can do it like this:
 
 ```javascript
-//app/config/environment.js
+// app/config/environment.js
 
 ENV['ember-cli-workbox'] = {
   enabled: environment !== 'test',
-
   debug: true,
-
   autoRegister: true,
-
-  importScriptsTransform(importScripts) {
-    return importScripts.map((importScript) => `https://example-cdn.com/${importScript}`);
-  },
-
   importScriptsGlobPatterns: [
     'assets/service-workers/*.js'
   ]
 };
 ```
 
+```javascript
+// ember-cli-build.js
+const app = new EmberAddon(defaults, {
+  importScriptsTransform(importScripts) {
```

Comment:
why not having a `workbox` property instead of defining the function in the root object? Something like:

```js
const app = new EmberAddon(defaults, {
  workbox: {
    importScriptsTransform
  }
});
```